reframe/core/pipeline.py

```diff
@@ -670,6 +678,7 @@ def _rfm_init(self, name=None, prefix=None):
         self.num_tasks_per_socket = None
         self.use_multithreading = None
         self.exclusive_access = False
+        self.max_queuing_time = None
```

Comment:
What if we called the variable `max_queue_time`. "Queuing" and "queueing" are both correct spellings, so I am sure it will create confusion.